webapp/tests/karma/ts/services/db.service.spec.ts

```diff
@@ -234,8 +234,14 @@ describe('Db Service', () => {
   });
 
   describe('method wrapping', () => {
+    let testDB;
+    let syncTarget;
+
     beforeEach(() => {
       window.PouchDB = require('pouchdb-browser').default;
```

Comment:
All these functions from L251 (bulkGet, get, allDocs, etc) aren't located in `window.PouchDB.prototype`. The PouchInternal's constructor calls `_setup()` to add all those functions to a DB instance.  
To make these tests work, we create an instance and make `window.PouchDB` return it, so later we stub its functions.